frontend/webpack.conf.js

```diff
@@ -21,11 +25,11 @@ module.exports = function(debug) { return {
     },
 
     module: {
-        loaders: [
+        rules: [
```

Comment:
`loaders` are `rules` now https://webpack.js.org/guides/migrating/#module-loaders-is-now-module-rules